diff --git a/node_modules/@polkadot-api/cli/dist/main/src/commands/generate.js b/node_modules/@polkadot-api/cli/dist/main/src/commands/generate.js
index 602eb43..89858d2 100644
--- a/node_modules/@polkadot-api/cli/dist/main/src/commands/generate.js
+++ b/node_modules/@polkadot-api/cli/dist/main/src/commands/generate.js
@@ -318,6 +318,7 @@ async function compileCodegen(packageDir) {
       emitDeclarationOnly: true,
       target: "esnext",
       module: "esnext",
+      moduleResolution: "bundler",
       resolveJsonModule: true,
       allowSyntheticDefaultImports: true,
       outDir
diff --git a/node_modules/@polkadot-api/cli/dist/src/commands/generate.js b/node_modules/@polkadot-api/cli/dist/src/commands/generate.js
index 602eb43..89858d2 100644
--- a/node_modules/@polkadot-api/cli/dist/src/commands/generate.js
+++ b/node_modules/@polkadot-api/cli/dist/src/commands/generate.js
@@ -318,6 +318,7 @@ async function compileCodegen(packageDir) {
       emitDeclarationOnly: true,
       target: "esnext",
       module: "esnext",
+      moduleResolution: "bundler",
       resolveJsonModule: true,
       allowSyntheticDefaultImports: true,
       outDir
